custom_attributes.go

```diff
@@ -0,0 +1,169 @@
+package gitlab
+
+import (
+	"fmt"
+)
+
+// CustomAttributesService handles communication with the group, project and
+// user custom attributes related methods of the GitLab API.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/custom_attributes.html
+type CustomAttributesService struct {
+	client *Client
+}
+
+// CustomAttribute struct is used to unmarshal response to api calls.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/custom_attributes.html
+type CustomAttribute struct {
+	Key   string `json:"key"`
+	Value string `json:"value"`
+}
+
+// ListCustomUserAttributes lists the custom attributes of the specified user.
+//
+// GitLab API docs:  https://docs.gitlab.com/ce/api/custom_attributes.html#list-custom-attributes
```

Comment:
I try to keep comments below 80 chars. When the URL is long and exceeds 80 chars, then just put it on a line of its own like this:

```
// GitLab API docs: 
// https://docs.gitlab.com/ce/api/custom_attributes.html#list-custom-attributes
```

For code I try to keep a limit of 100 chars, except for function signatures which I don't limit at all.